clients/spark/core/src/main/scala/io/treeverse/clients/StorageUtils.scala

```diff
@@ -153,15 +153,9 @@ object StorageUtils {
 
     private def getAWSS3Region(client: AmazonS3, bucket: String): String = {
       var request = new GetBucketLocationRequest(bucket)
-      request = request.withSdkClientExecutionTimeout(TimeUnit.SECONDS.toMillis(15).intValue())
+      request = request.withSdkClientExecutionTimeout(TimeUnit.SECONDS.toMillis(1).intValue())
       val bucketRegion = client.getBucketLocation(request)
-      val region = Region.fromValue(bucketRegion)
-      // The comparison `region.equals(Region.US_Standard))` is required due to AWS's backward compatibility:
-      // https://github.com/aws/aws-sdk-java/issues/1470.
-      // "us-east-1" was previously called "US Standard". This resulted in a return value of "US" when
-      // calling `client.getBucketLocation(bucket)`.
-      if (region.equals(Region.US_Standard)) "us-east-1"
-      else region.toString
+      Region.fromValue(bucketRegion).toAWSRegion().getName()
```

Comment:
What if region returns `US Standard`?
Will it still work?

Reply:
Yes, in this case:
1. Region.fromValue will return [`com.amazonaws.services.s3.model.Region.US_Standard`](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/s3/model/Region.html#US_Standard).
2. Then, calling `toAWSRegion` will return ```RegionUtils.getRegion("us-east-1")```.

Reply:
but then getName will return `US` again, no?

Reply:
No: from the [docs](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/regions/Region.html#getName--) of `com.amazonaws.regions.Region`:
```
The unique system ID for this region; ex: "us-east-1".
```

(also tested manually)